examples/04_glm_first_level/plot_bids_features.py

```diff
@@ -193,60 +193,132 @@
+    / "nilearn_glm"
+    / "sub-10159"
+    / (
+        "sub-10159_task-stopsignal_space-MNI152NLin2009cAsym_"
+        "contrast-stopsuccessMinusGo_stat-z_statmap.nii.gz"
+    )
+)
+
 plot_glass_brain(
     z_map,
-    threshold=norm.isf(0.001),
+    threshold=stat_threshold,
     plot_abs=False,
     display_mode="z",
     figure=plt.figure(figsize=(4, 4)),
 )
 show()
 
 # %%
-# We can get a latex table from a Pandas Dataframe for display and publication
-# purposes
+# The saved results include a table of activated clusters.
+#
+# .. note::
+#
+#     This table can also be generated by using the function
+#     :func:`nilearn.reporting.get_clusters_table`.
+#
+#     .. code-block:: python
+#
+#        from nilearn.reporting import get_clusters_table
+#
+#        table = get_clusters_table(
+#            z_map,
+#            stat_threshold=norm.isf(0.001),
+#            cluster_threshold=10
+#        )
 #
 # .. seealso::
 #
-#     This function does not report any named anatomical location
-#     for the clusters.
+#     The restults saved to disk and the output of get_clusters_table
+#     do not contain the anatomical location of the clusters.
 #     To get the names of the location of the clusters
 #     according to one or several atlases,
 #     we recommend using
 #     the `atlasreader package <https://github.com/miykael/atlasreader>`_.
 #
-from nilearn.reporting import get_clusters_table
+import pandas as pd
 
-table = get_clusters_table(z_map, norm.isf(0.001), 10)
-print(table.to_latex())
+table_file = (
+    output_dir
+    / "derivatives"
+    / "nilearn_glm"
+    / "sub-10159"
+    / (
+        "sub-10159_task-stopsignal_space-MNI152NLin2009cAsym_"
+        "contrast-stopsuccessMinusGo_clusters.tsv"
+    )
+)
+
+table = pd.read_csv(table_file, sep="\t")
+
+table
 
 # %%
-# Saving model outputs to disk
-# ----------------------------
+# We can get a latex table from a Pandas Dataframe
+# for display and publication purposes.
 #
-# We can now easily save the main results,
-# the model metadata and an HTML report to the disk.
+# .. note::
 #
-output_dir = Path.cwd() / "results" / "plot_bids_features"
-output_dir.mkdir(exist_ok=True, parents=True)
-
-from nilearn.interfaces.bids import save_glm_to_bids
-
-save_glm_to_bids(
-    model,
-    contrasts="StopSuccess - Go",
-    contrast_types={"StopSuccess - Go": "t"},
-    out_dir=output_dir / "derivatives" / "nilearn_glm",
-)
+#   This requires to have jinja2 installed:
+#
+#     .. code-block:: bash
+#
+#        pip install jinja2
+#
+print(table.to_latex())
 
-# %%
-# View the generated files
-files = sorted((output_dir / "derivatives" / "nilearn_glm").glob("**/*"))
-print("\n".join([str(x.relative_to(output_dir)) for x in files]))
+#  %%
+#   You can also print the output to markdown,
+#   if you have the `tabulate` dependencies installed.
+#
+#     .. code-block:: bash
+#
+#        pip install tabulate
+#
+#     .. code-block:: python
+#
+#        table.to_markdown()
```

Comment:
This renders as a comment in a cell right now. Maybe better to put it in an admonition box?

Reply:
![Screenshot 2025-04-01 at 1 18 15 PM](https://github.com/user-attachments/assets/621b2a5d-34e9-4df0-a2cd-738da73d7a2d)

Reply:
I think I had messed up some indent